pallets/subtensor/src/lib.rs

```diff
@@ -1536,42 +1536,11 @@ pub mod pallet {
 			Self::do_set_total_issuance(origin, total_issuance)
 		}
 
-		#[pallet::call_index(50)]
+		#[pallet::call_index(47)]
```

Comment:
```suggestion
		#[pallet::call_index(50)]
```

We shouldn't change the call indices for existing calls

Reply:
huum, right. fixed.